PostgresAssignment/package.json

```diff
@@ -0,0 +1,347 @@
+{
+  "name": "school_postgres_api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/startServer.ts",
+  "dependencies": {
+    "accepts": "^1.3.7",
```

Comment:
why do we have so many dependencies ? Anticipating to use them in future ?
Can we have only those dependencies required to run this project.

Reply:
Yes i will remove the unwanted dependencies